packages/rancher-vsphere/rancher-vsphere-cpi/generated-changes/patch/Chart.yaml.patch

```diff
@@ -1,11 +1,11 @@
 --- charts-original/Chart.yaml
 +++ charts/Chart.yaml
-@@ -8,7 +8,7 @@
-   catalog.cattle.io/rancher-version: '>= 2.9.0-0'
+@@ -5,7 +5,7 @@
+   catalog.cattle.io/namespace: kube-system
+   catalog.cattle.io/os: linux
+   catalog.cattle.io/permits-os: linux,windows
+-  catalog.cattle.io/rancher-version: '>= 2.9.0-0'
++  catalog.cattle.io/rancher-version: '>= 2.10.0-0'
    catalog.cattle.io/release-name: vsphere-cpi
  apiVersion: v1
--appVersion: 1.30.1
-+appVersion: 1.8.0
- description: vSphere Cloud Provider Interface (CPI)
- icon: https://charts.rancher.io/assets/logos/vsphere-cpi.svg
- keywords:
+ appVersion: 1.31.0
```

Comment:
Shouldn't this equal the `version` of the CPI chart, `1.9.1`? That's the convention the prior version of this chart used with `1.8.0`.

Reply:
I believe the value was set incorrectly in the past, as the `appVersion` should reflect the version of the application that the chart contains, according to the [Helm documentation](https://helm.sh/docs/topics/charts/#the-chartyaml-file). In this case, the upstream chart's value of 1.30.1 is also outdated, so I'll need to update it to 1.31.0.

Reply:
I think keeping it as `1.9.1` would be better, as it allows us to track minor changes to the chart which do not require  a bump to the appVersion. For instance, changing `questions.yaml`